src/test/ui/terminal-width/tabs-trimming.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0408]: variable `v` is not bound in all patterns
+  --> $DIR/tabs-trimming.rs:9:16
+   |
+LL | ...   v @ 1 | 2 | 3 => panic!("You gave me too little money {}", v), // Long text here: TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT...
```

Comment:
Is 3 spaces at the front the correct behavior?

Reply:
I am not 100% confident on that, but it is a natural result of the current algorithm subtracting off leading whitespace when there is a lot of it. It also matches @est31’s expected output in the issue.

Reply:
Yes, I think it makes sense given the "leading whitespace removal" algorithm.